suricata.yaml.in

```diff
@@ -469,6 +469,8 @@ af-packet:
     #rollover: yes
     # To use the ring feature of AF_PACKET, set 'use-mmap' to yes
     use-mmap: yes
+    # Use tpacket_v3, capture mode, only active if user-mmap is true
+    #tpacket-v3: true
```

Comment:
Maybe it will not be available on really old kernel/distro. I would not be opposed to set it to true in new configuration file so we keep old user to v2 and switch new one to v3. Pushing that in new PR.